main.py

```diff
@@ -1,10 +1,64 @@
+from typing import Any
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
+def race_func(player: dict) -> Any:
+    if not Race.objects.filter(name=player["race"]["name"]).exists():
+        race_inst, created = Race.objects.get_or_create(
+            name=player["race"]["name"],
+            defaults={"description": player["race"]["description"]}
+        )
+        return race_inst
+
+
+def skill_func(player: dict) -> None:
+    for skill in player["race"]["skills"]:
+        if not Skill.objects.filter(name=skill["name"]).exists():
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                defaults={"bonus": skill["bonus"], "race": race_func(player)}
+            )
+
+
+def guild_func(player: dict) -> Any:
+    guild = player.get("guild")
+    if guild is not None:
+        if not Guild.objects.filter(name=player["guild"]["name"]).exists():
```

Comment:
remove this check no need
Think, why you use this? 
get_or_create will create new or get already existed
But you do: check if exist and than  create new or get already existed